package.json

```diff
@@ -21,26 +21,28 @@
   },
   "homepage": "https://github.com/Polymer/polymer-build#readme",
   "dependencies": {
-    "@types/fs-extra": "0.0.32",
+    "@types/parse5": "^2.2.32",
     "@types/vinyl": "^1.1.29",
     "@types/vinyl-fs": "0.0.28",
-    "dom5": "^1.3.1",
-    "fs-extra": "^0.30.0",
+    "dom5": "^2.0.1",
     "hydrolysis": "^1.23.3",
     "minimatch-all": "^1.0.2",
-    "multipipe": "^0.3.0",
+    "multipipe": "^1.0.2",
+    "parse5": "^2.2.2",
     "plylog": "^0.4.0",
-    "polymer-analyzer": "2.0.0-alpha.10",
+    "polymer-analyzer": "^2.0.0-alpha.17",
```

Comment:
Ah good call. FWIW I don't think `^` is doing anything in npm when you have special `-alpha` tags like this one.